lib/ldap_fluff/generic_member_service.rb

```diff
@@ -70,7 +70,7 @@ def get_logins(userlist)
 
   def get_login_from_entry(entry)
     [@attr_login, 'uid', 'cn'].each do |attribute|
-      return entry.send(attribute) if entry.respond_to? attribute
+      return entry.send(attribute.map(&:downcase)) if entry.respond_to? attribute
```

Comment:
Yeah, which is why I'd prefer to fix it in Foreman, since we could have it work fully case insensitively there, then remove the last downcases here.  Adding another downcase here could break it for people using the AD support today in a case sensitive manner.